exercises/clock/canonical-data.json

```diff
@@ -213,7 +213,12 @@
           "minute": 1,
           "add": 3500,
           "expected": "11:21"
-        },
+        }
+      ]
+    },
+    {
+      "description": "Subtract minutes",
+      "cases": [
         {
           "description": "subtract minutes",
           "property": "add",
```

Comment:
I agree that `add` isn't the best property name, and maybe `adjust` would be a better choice. On the other hand, this is only necessarily visible to track maintainers, as they can choose more descriptive names when implementing the exercise.

If I where to choose, I would merge this the way it is and leave other MAJOR changes for another PR, because maybe it deserves a rewriting. I don't understand what the `create` property is testing at all...